content/en/docs/concepts/services-networking/network-policies.md

```diff
@@ -94,7 +94,7 @@ __egress__: Each NetworkPolicy may include a list of allowed `egress` rules.  Ea
 So, the example NetworkPolicy:
 
 1. isolates "role=db" pods in the "default" namespace for both ingress and egress traffic (if they weren't already isolated)
-2. (Ingress rules) allows connections to all pods in the “default” namespace with the label “role=db” on TCP port 6379 from:
+2. (Ingress rules) allows connections to all pods in the "default" namespace with the label "role=db" on TCP port 6379 from:
```

Comment:
nit: I would check the style guide. I think namespaces and labels use emphasis not double quotes?

Reply:
It seems that several places use ``` and others use `"` as a quote. Which one is preferred?

Reply:
Hi @shuuji3 .
I am not entirely sure. Generally, I would not style (code style) the string key=value (The key=value pairs ...).
I searched and found that "key=value" appears in some of the reference pages and `key=value` appears in the troubleshooting page.
Use code style when this pair represents actual keys used with a command flag, such as:
`kube-apiserver --client-ca-file=FILENAME`. The entire command and flags are contained in code style.

Reply:
@kbhawkey Thanks for the investigation and explanation. I see... So for this PR, can we keep the quote characters as-is for now and leave it for another fix? Because we don't know the quick decision and it requires additional commits (also, this PR covers many files and requires frequent rebase.😅).